fhireval/test_suite/concept_map/test_conceptmap_create.py

```diff
@@ -1,14 +1,15 @@
 import pytest
 import pdb
 
-from fhireval.test_suite.concept_map import example_code_system_source, reset_testdata
+from fhireval.test_suite.crud import prep_server
+from copy import deepcopy
 
-test_id = f"{'2.8.1':<10} - Create ConceptMap"
+test_id = f"{'2.4.1':<10} - Create ConceptMap"
 
 test_weight = 2
 
-def test_codemap_create(host):
-    reset_testdata(host)
+def test_conceptmap_create(host, prep_server):
+    example_code_system_source = prep_server['CMG']['ConceptMap'][0]
     result = host.post('ConceptMap', example_code_system_source, validate_only=False)
     assert result['status_code'] == 201
```

Comment:
For some systems, a POST result returns 200 and the following seems like a more robust way:

```Python
try:
    resp.raise_for_status()
except Exception as e:
    raise e
```

Reply:
HTTP's 201 means Created, so I'm not sure what else a success to a create situation would be other than a problematic response.